packages/extension-api/src/extension-api.d.ts

```diff
@@ -465,8 +465,27 @@ declare module '@podman-desktop/api' {
   }
 
   export namespace commands {
+    /**
+     * Define a command, to be executed later, either by calling {@link commands.executeCommand} or by referencing its name in the `command` field of a {@link StatusBarItem}.
+     *
+     * @param command the name of the command. The name must unique over all extensions. It is recommended to prefix this name with the name of the extension, to avoid conflicts with commands from other extensions.
```

Comment:
```suggestion
     * @param command the name of the command. The name must be unique over all extensions. It is recommended to prefix this name with the name of the extension, to avoid conflicts with commands from other extensions.
```